doc/source/tutorial-quickstart-xgboost.rst

```diff
@@ -144,8 +145,8 @@ as below:
         new_data = xgb.DMatrix(x, label=y)
         return new_data
 
-The ClientApp
--------------
+ClientApp
```

Comment:
`The ClientApp` with `The` just follows the same thing in PyTorch tutorial - https://flower.ai/docs/framework/tutorial-quickstart-pytorch.html. Do we want remove it? ;) It's just a minor thing though

Reply:
Oh I see. Bringing `The` back!